packages/rancher-kiali-server/rancher-kiali-server.patch

```diff
@@ -37,9 +37,45 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/C
 +  catalog.cattle.io/requires-gvr: prometheuses.monitoring.coreos.com/v1
 +  catalog.rancher.io/namespace: cattle-istio-system
 +  catalog.rancher.io/release-name: rancher-kiali-server
+diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/templates/_helpers.tpl packages/rancher-kiali-server/charts/templates/_helpers.tpl
+--- packages/rancher-kiali-server/charts-original/templates/_helpers.tpl
++++ packages/rancher-kiali-server/charts/templates/_helpers.tpl
+@@ -17,10 +17,7 @@
+   {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+ {{- else }}
+   {{- $name := default .Chart.Name .Values.nameOverride }}
+-  {{- if contains $name .Release.Name }}
+-    {{- .Release.Name | trunc 63 | trimSuffix "-" }}
+-  {{- else }}
+-    {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
++    {{- printf "%s" $name | trunc 63 | trimSuffix "-" }}
```

Comment:
Reached out to @brendarearden offline about this, but we should consider if there are any consequences to using `.Chart.Name` instead of `.Release.Name` here since `rancher-istio` may need to support multiple versions of the same chart being in the same cluster in the future (which would conflict if resources have the same name based on the `.Chart.Name` approach).

Reply:
Bundling Kiali at all kills/complicates the idea of canary in our istio installer. The kiali-server chart does not support canary deployments, we need to build that in. At that point we would need to add the revision to the name here (but then removal of previous version going forward has to be manual), or just say kiali will upgrade immediately to newest version.